applications/app.c

```diff
@@ -88,13 +88,13 @@ void app_set_configuration(app_configuration *conf) {
 		break;
 
 	case APP_PPM_UART:
-		hw_stop_i2c();
+        hw_stop_i2c();
```

Comment:
Still spaces

Reply:
Do you know a good tool, to check the indent ?

Reply:
there are still spaces :-)